Workbooks/UpdateCompliance/UpdateComplianceHub/Update Compliance Hub.workbook

```diff
@@ -554,10 +554,18 @@
                       {
                         "type": 1,
                         "content": {
-                          "json": "** Related links **\r\n\r\nNow generally available: <a href=\"https://techcommunity.microsoft.com/t5/windows-it-pro-blog/now-generally-available-windows-update-for-business-reports/ba-p/3677018\">Windows Update for Business reports </a>\r\n\r\nKeep your Windows client devices up to date with <a href=\"https://docs.microsoft.com/windows/deployment/update/waas-manage-updates-wufb?WT.mc_id=Portal-fx\">Windows Update for Business</a>\r\n\r\nManage Driver Updates with the <a href=\"https://techcommunity.microsoft.com/t5/windows-it-pro-blog/commercial-driver-and-firmware-servicing-is-publicly-available/ba-p/3741194\">Windows Update for Business deployment service</a>\r\n\r\nNew capability! <a href=\"https://techcommunity.microsoft.com/t5/windows-it-pro-blog/windows-update-for-business-reports-access-and-region-control/ba-p/3793118\">Learn about flexible data routing and region control for devices in your environment</a>"
+                          "json": "** Related links **\r\n\r\nNow generally available: <a href=\"https://techcommunity.microsoft.com/t5/windows-it-pro-blog/now-generally-available-windows-update-for-business-reports/ba-p/3677018\">Windows Update for Business reports </a>\r\n\r\nKeep your Windows client devices up to date with <a href=\"https://docs.microsoft.com/windows/deployment/update/waas-manage-updates-wufb?WT.mc_id=Portal-fx\">Windows Update for Business</a>\r\n\r\nManage Driver Updates with the <a href=\"https://techcommunity.microsoft.com/t5/windows-it-pro-blog/commercial-driver-and-firmware-servicing-is-publicly-available/ba-p/3741194\">Windows Update for Business deployment service</a>"
                         },
                         "customWidth": "25",
                         "name": "RelatedLinks"
+                      },
+                      {
+                        "type": 1,
+                        "content": {
+                          "json": "** Windows 11 **\r\n\r\nGet started with a Windows 11 Pilot today using feature targeting in Intnue or the management tool of your choice. Learn how to create a <a href=\"https://learn.microsoft.com/en-us/windows/whats-new/windows-11-plan#deployment-planning\">deployment plan</a>.\r\n\r\nInterested in seeing if your critical business applications work on Windows 11? Check out our cloud validation service, <a href=\"https://aka.ms/WufBOnRamp\">Test Base</a>.\r\n\r\nRead about how Microsoft successfully rolled out Windows 11 to 190,000 devices. <a href=\"https://www.microsoft.com/insidetrack/blog/unpacking-microsofts-speedy-upgrade-to-windows-11/\">Microsoft's speedy upgrade to Windows 11</a>.\r\n\r\nNeed help? Eligible customers may also request deployment assistance from <a href=\"https://aka.ms/FastTrackWindowsEnt\">FastTrack</a>."
```

Comment:
```suggestion
                          "json": "** Windows 11 **\r\n\r\nGet started with a Windows 11 Pilot today using feature targeting in Intnue or the management tool of your choice. Learn how to create a <a href=\"https://learn.microsoft.com/en-us/windows/whats-new/windows-11-plan#deployment-planning\">deployment plan</a>.\r\n\r\nInterested in seeing if your critical business applications work on Windows 11? Check out our cloud validation service, <a href=\"https://aka.ms/WufBOnRamp\">Test Base</a>.\r\n\r\nRead about how Microsoft successfully rolled out Windows 11 to 190,000 devices. <a href=\"https://www.microsoft.com/insidetrack/blog/unpacking-microsofts-speedy-upgrade-to-windows-11/\">Microsoft's speedy upgrade to Windows 11</a>.\r\n\r\nNeed help? Eligible customers may also request deployment assistance from <a href=\"https://aka.ms/FastTrackWindowsEnt\">FastTrack</a>."
```

Happned to see this while I was here looking at what I need to doc for this. Stripping out language code from URL so docs picks it up from user's prefrence instead.